dash/_patch.py

```diff
@@ -99,6 +99,9 @@ def __ior__(self, other):
         self.update(E=other)
         return _noop
 
+    def __iter__(self):
+        raise TypeError("Iterating over patch objects is prohibited.")
```

Comment:
```suggestion
        raise TypeError("Patch objects are write-only, you cannot iterate them.")
```
Just to make clear _why_ it's prohibited.